docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -222,13 +221,22 @@ export default class MasonryContainer extends Component<Props, State> {
     }
 
     this.randomNumberSeed += 1;
-    const newItems = generateExampleItems({
-      name,
-      total: until - from,
-      from,
-      baseHeight,
-      randomNumberSeed: this.randomNumberSeed,
-    });
+    const newItems =
+      pinHeightsSample && pinHeightsSample.length > 1
+        ? generateRealisticExampleItems({
+            name,
+            numberOfItems: until - from,
+            previousItemCount: from,
+            randomNumberSeed: Math.random(),
+            pinHeightsSample,
+          })
+        : generateExampleItems({
+            name,
+            numberOfItems: until - from,
+            previousItemCount: from,
+            baseHeight,
+            randomNumberSeed: this.randomNumberSeed,
```

Comment:
ditto on having real and non-real, and how come we use Math.random in one place, and the seed in another?